File: create_dirs.py
import os

# Create directories
dir1 = r"e:\nodejs\billing-internet\src\app\pengeluaran"
dir2 = r"e:\nodejs\billing-internet\src\app\api\pengeluaran"

os.makedirs(dir1, exist_ok=True)
os.makedirs(dir2, exist_ok=True)

print("Directories created successfully")
print(f"✓ {dir1}")
print(f"✓ {dir2}")

File: create-dirs.js
const fs = require('fs');
const path = require('path');

const dirs = [
  'e:\\nodejs\\billing-internet\\src\\app\\pengeluaran',
  'e:\\nodejs\\billing-internet\\src\\app\\api\\pengeluaran'
];

dirs.forEach(dir => {
  fs.mkdirSync(dir, { recursive: true });
  console.log(`Created: ${dir}`);
});

console.log('All directories created successfully');
